x-pack/plugins/alerting/common/rule.ts

```diff
@@ -81,13 +81,13 @@ export interface RuleExecutionStatus {
 export type RuleActionParams = SavedObjectAttributes;
 export type RuleActionParam = SavedObjectAttribute;
 
-export interface RuleActionFrequency extends SavedObjectAttributes {
```

Comment:
I decided to remove them to have more control over the types and avoid bugs in the future where we spread properties that should not exist.